CMakeLists.txt

```diff
@@ -208,6 +208,7 @@ if(UNIX AND NOT APPLE) # i.e.: Linux
 
     if(BUILD_WSI_XCB_SUPPORT)
         find_package(XCB REQUIRED)
+        include_directories(SYSTEM ${XCB_INCLUDE_DIR})
```

Comment:
Does this addition fix build errors for OpenBSD? Would like a clear indication that this is necessary before merging it.

Reply:
Yes, it does.

```
[  5%] Building C object loader/CMakeFiles/asm_offset.dir/asm_offset.c.o
In file included from /home/brad/tmp/Vulkan-Loader/loader/asm_offset.c:26:
In file included from /home/brad/tmp/Vulkan-Loader/loader/loader_common.h:34:
/usr/local/include/vulkan/vulkan.h:53:10: fatal error: 'xcb/xcb.h' file not found
#include <xcb/xcb.h>
         ^~~~~~~~~~~
1 error generated.
```